chromium_src/chrome/browser/ssl/https_only_mode_upgrade_interceptor.cc

```diff
@@ -5,8 +5,34 @@
 
 #include "chrome/browser/ssl/https_only_mode_upgrade_interceptor.h"
 
+#include "brave/browser/brave_browser_process.h"
+#include "brave/components/brave_shields/browser/brave_shields_util.h"
+#include "chrome/browser/content_settings/host_content_settings_map_factory.h"
+#include "chrome/browser/profiles/profile.h"
+#include "components/prefs/pref_service.h"
 #include "net/base/url_util.h"
 
+class GURL;
+
+namespace content {
+class BrowserContext;
+}  // namespace content
+
+namespace {
+
+bool ShouldUpgradeToHttps(content::BrowserContext* context, const GURL& url) {
```

Comment:
Effectively this function is already returning false because it calls brave_shields::ShouldUpgradeToHttps, but I will add an extra check here to return earlier.